packages/v4/src/content/design-guidelines/components/label/label.md

```diff
@@ -46,7 +46,7 @@ Optionally add an icon to your label in cases where additional visual informatio
 
```

Comment:
I wonder whether it would be helpful for this example to show same color labels with different icons used to distinguish them. At least that's what I expected after reading the comment above.

Reply:
That's fair. Does it make sense to include both scenarios? I can definitely mock up an example with the same color labels and different icons.

Reply:
Sure. I think you could have two examples.

Reply:
Added!

Reply:
@bmignano I don't think that this is what you intended, but here's how this looks when I preview the site:

![Screen Shot 2021-01-12 at 4 56 42 PM](https://user-images.githubusercontent.com/6619860/104379432-74e8c800-54f7-11eb-812e-0eddac7e2eee.png)

Maybe place this sentance, "You can also use them to distinguish labels of the same color if needed." between the two image tags and that will make them stack.

Reply:
Ah, thanks for catching that @mcarrano ! Updated with your suggestion.